eos-migrate-image-defaults

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -ex
+
+old_path=/var/eos-image-defaults
+new_path=/var/lib/eos-image-defaults
+dconf_profile=/etc/dconf/profile/user
+
+if [ -d ${old_path} -a ! -h ${old_path} -a ! -e ${new_path} ]; then
+  mv ${old_path} ${new_path}
+fi
+
+if [ ! -e ${old_path} -a -d ${new_path} ]; then
+  ln -s ${new_path} ${old_path}
+fi
+
+# unfortunately the /etc/dconf/profile/user is not contained in the
+# ostree, so must be manually updated when this migration is made
```

Comment:
I don't think we use this file anymore since we moved to the new GDM that installs it in `/usr`? The overrides created by the image builder are in a subdirectory of the image-defaults location AFAIR, so what else is in this file?